Foundation.hs

```diff
@@ -147,10 +152,22 @@ instance YesodAuth App where
                 }
 
     -- You can add other plugins like Google Email, email or OAuth here
-    authPlugins _ = [authOpenId Claimed []]
+    authPlugins app = [authOpenId Claimed []] ++ extraAuthPlugins
+      -- Enable authDummy login when in development mode.
+      where extraAuthPlugins = [authDummy | appDevelopment $ appSettings app]
+
+
 
     authHttpManager = getHttpManager
 
+-- | Access function to determine if a user is logged in.
+isAuthenticated :: Handler AuthResult
+isAuthenticated = do
+    mu <- maybeAuthId
```

Comment:
I agree, I wonder if there's a more standard definition for this. We could also go with `mAuthId <- maybeAuthId ` which is more explicit.